cli/util/cancel.go

```diff
@@ -69,6 +69,18 @@ func cancelTx(ctx *cli.Context) error {
 	if err != nil {
 		return cli.NewExitError(fmt.Errorf("failed to send conflicting transaction: %w", err), 1)
 	}
+	if ctx.Bool("await") {
+		resTx, _ := c.GetRawTransactionVerbose(resHash)
```

Comment:
You don't need this `GetRawTransaction` request, because it will send one more unnecessary RPC request to the node. You actually have everything that needed to perform awaiting: hash of the main transaction (if it's available), hash of the conflicting transaction (`resHash`, always available), VUB value of the conflicting transaction (it's the second return value of SendTunedRun) (or maximum from VUB and VUB of main transaction if main transaction is available). You need to await for one of these two transactions, thus use `WaitAny` API.